src/main/java/Hint.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Hint {
+
+    public int countStrike(String number, String input) {
+        List<String> numberList = Arrays.asList(number.split(""));
+        List<String> inputList = Arrays.asList(input.split(""));
+        return (int)IntStream.range(0, numberList.size())
+            .filter(index -> numberList.get(index).equals(inputList.get(index)))
+            .count();
+    }
+
+    public int countBall(String number, String input) {
+        List<String> numberList = Arrays.asList(number.split(""));
+        List<String> inputList = Arrays.asList(input.split(""));
+        int containCount = (int)inputList.stream()
+            .filter(item -> numberList.contains(item))
+            .count();
+        int StrikeCount = (int)IntStream.range(0, numberList.size())
```

Comment:
```suggestion
        int strikeCount = (int)IntStream.range(0, numberList.size())
```

일반적으로 [자바 코드 컨벤션](http://kwangshin.pe.kr/blog/java-code-conventions-%ec%9e%90%eb%b0%94-%ec%bd%94%eb%94%a9-%ea%b7%9c%ec%b9%99/)에서는 변수명의 시작은 소문자로 시작하고 [카멜 표기법](https://dpdpwl.tistory.com/55)을 사용할 것을 권장하고 있습니다.

Reply:
오타 체크를 하지 못한 것 같습니다. 수정하여 반영하겠습니다.